pkg/client/mapiserver/snapshot.go

```diff
@@ -70,18 +69,9 @@ func CreateSnapshot(volName string, snapName string) error {
 	}
 	defer resp.Body.Close()
 
-	body, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		return err
-	}
-
 	code := resp.StatusCode
-	if err == nil && code != http.StatusOK {
```

Comment:
This check required here in case of err is nil but there is err return in a body

Reply:
Done